examples/examples_test.go

```diff
@@ -13,27 +13,19 @@ func getBaseOptions(t *testing.T) integration.ProgramTestOptions {
 	return integration.ProgramTestOptions{
 		ExpectRefreshChanges: true,
 		Secrets: map[string]string{
-			"github:token":        token,
-			"github:organization": org,
+			"github:token": token,
+			"github:owner": org,
 		},
 	}
 }
 
 func getGitHubTestingOrg(t *testing.T) string {
 	org := os.Getenv("GH_ORGANIZATION")
-	if org == "" {
-		t.Skipf("Skipping test due to missing GH_ORGANIZATION environment variable")
```

Comment:
What about local testing? Is there to way to only fail when we're running on Github?

Reply:
I'm not sure I entirely understand your question - these variables are the CI equivalent to `GITHUB_OWNER` and `GITHUB_TOKEN` from here: https://www.pulumi.com/registry/packages/github/installation-configuration/.